ui/app/src/views/LatestRuleset/LatestRuleset.vue

```diff
@@ -1,4 +1,134 @@
 <template>
-  <div>
-  </div>
+  <v-container id="consult">
+    <h1>{{path}}</h1>
+
+    <v-layout>
+      <v-flex md2 mt-5>
+        <v-toolbar color="grey" dark>
+          <v-toolbar-title>Parameters</v-toolbar-title>
+        </v-toolbar>
+        <v-card class="height-card scroll">
+          <v-card-text v-for="param in ruleset.signature.params" :key="param.name">{{param.name}}: {{param.type}}</v-card-text>
+        </v-card>
+      </v-flex>
+
+      <v-flex md2 ma-5>
+        <v-toolbar color="grey" dark>
+          <v-toolbar-title>Return type</v-toolbar-title>
+        </v-toolbar>
+        <v-card class="height-card">
+          <v-card-text>{{ruleset.signature.returnType}}</v-card-text>
+        </v-card>
+      </v-flex>
+
+      <v-flex md2 ma-5>
+        <v-toolbar color="grey" dark>
+          <v-toolbar-title>Versions</v-toolbar-title>
+        </v-toolbar>
+        <v-card class="height-card scroll">
+          <v-card-text v-for="version in ruleset.versions" :key="version">{{version}}</v-card-text>
+        </v-card>
+      </v-flex>
+
+      <v-flex md3></v-flex>
+
+      <v-flex md3 mt-5>
+        <!-- change the link to edit page -->
+        <router-link to="/rulesets/new">
+          <v-btn
+            dark
+            color="primary"
+          >
+            Edit
+          </v-btn>
+        </router-link>
+      </v-flex>
+
+    </v-layout>
+
+    <!-- delegate rules to Rules component -->
+    <Rules v-model="ruleset" :editMode="false" />
+  </v-container>
 </template>
+
+<script>
+import axios from 'axios';
+import { Ruleset, Rule, Signature, Param } from '../NewRuleset/ruleset';
+import Rules from '../NewRuleset/Rules.vue';
+
+export default {
+  components: {
+    Rules,
+  },
+
+  props: {
+    path: {
+      type: String,
+    }
+  },
+
+  data() {
+    return {
```

Comment:
Because JS, and particularly Vue, is not our "normal" work, I think we should try to be especially diligent about adding comments / docstrings to methods.  What seems obvious today, might be obscure in a months time.

Reply:
I agree but for this one I think that it's not necessary to add comment. 
The `data` property is the main important things to know in Vue, it's for that reason that I didn't add comment, for example, it's like adding comment on top of the `main` function in a Go program.